packages/compartment-mapper/test/fixtures-assets/main.js

```diff
@@ -0,0 +1,26 @@
+import text from './text.text';
+import bytes from './bytes.bytes';
+import uint32 from './uint32.uint32';
```

Comment:
very nice.

I wonder if there's a straightforward way to link from docs to test examples such as this one.

Reply:
You mean like this?

https://github.com/endojs/endo/blob/acc828cf74308b7cdcf000f7492a53e3135bdfd3/packages/compartment-mapper/test/fixtures-assets/main.js#L1-L3

It should work in markdown files too. worth trying